### DISCUSSION TITLE
Add button to enter details manually

### DISCUSSION THREAD

**Discussion by OliDMX** [ente-io/ente#1040] [Enhancements]
In the past weeks i've changed all my 2factors to ente auth. I had a few times the problem that ente couldn't recognize the QR Code, so they might be too complicated for ente. 
But then i had to get the Code for 2Factor from my PC to my phone, without typing it off and just copying it.
So i had the idea if you could add an + Button like in the App for the Website, where you can enter the details manually.
Would be really nice!

**Comment by ua741** [comment]
> I had a few times the problem that ente couldn't recognize the QR Code,
This is strange. Would be possible for you to share the website name or sample QR code?

**Comment by OliDMX** [comment]
> > I had a few times the problem that ente couldn't recognize the QR Code,
> > This is strange. Would be possible for you to share the website name or sample QR code?
Epic Games as an example. I have added over 40 2FAs in the past days, i can't remember specifics, i just know some don't work..
